training/index.html

```diff
@@ -28,20 +28,13 @@ <h1>OME Training</h1>
                 to OMERO on one day and more advanced analysis, developer or sysadmin features
                 on the next.
             </p>
+            <p>
+                Email <code>training-support@lists.openmicroscopy.org.uk</code> to discuss a workshop at your site.
```

Comment:
If we're definitely having an alias `training-support@openmicroscopy.org` it'll be easier to never mention the full `lists.openmicroscopy.org.uk` address, in case we need to add spam protection like with the idr list.

Reply:
@joshmoore can we have that alias  please ? I will remove the address as mentioned by @manics , just like if we have it already. Thank you